src/trans.c

```diff
@@ -1500,16 +1500,24 @@ Obj FuncON_KERNEL_ANTI_ACTION (Obj self, Obj ker, Obj f, Obj n) {
   UInt    deg, i, j, rank, len;
   Obj     out;
 
-  if (INT_INTOBJ(ELM_LIST(ker, LEN_LIST(ker))) == 0) {
+  assert(IS_LIST(ker));
+  assert(IS_INTOBJ(n));
```

Comment:
Well, I'll merge this as is then, and you can wonder about better checks or moving it to the package later; that's for another PR, anyway :-)